.circleci/config.yml

```diff
@@ -427,14 +419,9 @@ jobs:
           no_output_timeout: 30m
 
   spm-xcode-14-1:
-    <<: *base-job
-    parameters:
-      xcode_version:
-        type: string
-        default: '14.1.0'
-      install_swiftlint:
-        type: boolean
-        default: false
```

Comment:
these jobs didn't actually install swiftlint, even if you pass in true here, since that happens if you call install dependencies, so you could have thought that it should work and gone crazy trying to figure it out